spec/policies/budget_policy_spec.rb

```diff
@@ -13,8 +16,21 @@
   it { is_expected.to permit_edit_and_update_actions }
   it { is_expected.to permit_action(:destroy) }
 
-  it "includes budget in resolved scope" do
-    resolved_scope = described_class::Scope.new(user, Budget.all).resolve
-    expect(resolved_scope).to include(budget)
+  context "as a BEIS user" do
+    let(:user) { create(:beis_user) }
+
+    it "includes all budgets in the resolved scope" do
```

Comment:
Does this still ensure that only budgets that belong to the current activity are viewed, as opposed to budgets from all activities?

Reply:
Yeeees it might help if I implemented that 🤦‍♂ 😅